src/ComposerRequireChecker/FileLocator/LocateComposerPackageSourceFiles.php

```diff
@@ -36,12 +36,17 @@ function (array $sourceDirs, $sourceDir) {
         );
         return array_values(array_map(
             function (string $sourceDir) use ($packageDir) {
-                return $packageDir . '/' . ltrim($sourceDir, '/');
+                return $this->normalizePath($packageDir . '/' . ltrim($sourceDir, '/'));
             },
             $flattened
         ));
     }
 
+    private function normalizePath(string $path)
+    {
+        return str_replace(array('\\', '/'), DIRECTORY_SEPARATOR, $path);
```

Comment:
The second argument can just be `'/'`, which is recognized as directory separator for all platforms by the PHP engine